src/components/plots/errorBarPlot.ts

```diff
@@ -0,0 +1,47 @@
+module Plottable {
+export module Plot {
+  export class ErrorBar<X> extends AbstractXYPlot<X,number> {
```

Comment:
Verify that we actually want the name ErrorBar.  I'm mainly wondering if this idea is not specific to barPlots / the plot itself is not really a bunch of bars